sdk/cosmosdb/cosmos/src/diagnostics/CosmosDiagnostics.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getCosmosDiagnosticsToString, getdiagnosticsdurationMilliseconds } from "./Diagnostics"; //static?
+export class CosmosDiagnostic {
+  /**
+   * Regions contacted for this request.
+   */
+  getRegionsContacted() {}
```

Comment:
I assume when you finish the functional spec we will discuss ths separately - to me min bar for diagnostics is:
- Duration
- ActivityId
- cumulated RequestCharge
- Request timeline (similar to GatewayStatistics in Java) - so that it is possible to get some hints why a request was slow (connection acquistition, purely transit time)
- Request diagnostics (which should have information about retries and durations for each request/retry attempt). 

- This PR seems to address duration and regions contacted for successful and failed requests - adding timeline and request charge will also help to validate whether the "generic DiagnostcHeader[] approach is scalable enough or whether using a typed class would be preferrable - i lean towards the latte r- but hey, I am much more familiar with strongly typed languages like Java and .Net - so, definitely open to opinions what is more intuitive in Javascript

Reply:
done